improver_tests/nowcasting/utilities/test_FillRadarHoles.py

```diff
@@ -41,8 +40,9 @@
 from ...set_up_test_cubes import set_up_variable_cube
 
 
-def create_masked_rainrate_data():
-    """Create a masked data array of rain rates in mm/h"""
+@pytest.fixture(name="rainrate")
+def rainrate_fixture() -> Cube:
```

Comment:
@tjtg I notice that [type hints](https://docs.python.org/3/library/typing.html) have been used with these pytest fixtures Is this a general style to be following for pytest fixtures? I'm not sure if we've previously used type hints in IMPROVER.

Reply:
I used the type hints here as a way to include the type information, without expanding to a longer multiline docstring containing a `Returns` section. A `Returns` section would be close to a duplicate of the existing docstring summary, so it seemed unnecessary.

Pytest doesn't have any requirement for type hints, it was just a convenient way to express the information about the fixture. Having type hints may be helpful for other fixture functions, but I didn't intend using it here as a hard requirement for others to follow.

Reply:
OK, thanks for the explanation.